### ISSUE TITLE
Major drop in accuracy with minor change to train image folder

### ISSUE THREAD

**Issue by Adelost** [divamgupta/image-segmentation-keras#121]
I have trained a dataset of 1000 images and masks and 4 classes with resnet50_unet to 99.3% accuracy. If I remove a single image and corresponding mask out of the 1000 images from the training image and training mask folder, accuracy is dropped to 85%. Visually inspecting the generated masks also confirms that accuracy has had a major drop.

I have not used image augmentation nor a separate validation dataset so I suspect I have a lot of over-fitting. However, my understanding is that changing from 1000 to 999, of otherwise identical training images, should not have any affect on the training accuracy with or without overfitting. Is there anything that could explain this behavior?

If needed I can try to provide a reproducible example with more detailed instructions later. However, the code is mostly identical to the example "Using the python module" from the readme, but with a different dataset and resnet50_unet(n_classes=4, input_height=512, input_width=512)

**Comment by divamgupta** [comment]
I believe that there is no problem in loading of the data.
I would be interested in looking at a  reproducible example.
